.github/workflows/pr-pw-tests.yml

```diff
@@ -0,0 +1,48 @@
+name: PR Playwright Tests
```

Comment:
Maybe `playwright-v4`? 
as we know that this workflow will be for v4 checkups
Of `playwright-playground`?